typescript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+const fullName = 'Aaditya Desai';
+console.log('value of fullName:', fullName);
+const isCool = true;
+console.log('value of isCool:', isCool);
+const totalPets = 0;
+console.log('value of totalPets:', totalPets);
+console.log('typeof fullName:', typeof 'fullName');
+console.log('typeof isCool:', typeof true);
+console.log('typeof totalPets:', typeof 0);
```

Comment:
You'll want to use the variables for isCool and totalPets here and not the value. If you just use the value, what would happen if you updated isCool or totalPets? You would still only return the typeof true and typeof 0!